kubernetes/celestia-local/service.yml

```diff
@@ -7,6 +7,12 @@ spec:
   selector:
     app: astria-dev-cluster
   ports:
-    - name: node-svc
+    - name: bridge-rest
```

Comment:
If we can remove gateway flags we can remove this port

Reply:
See note about using the rest api below